hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvokeOnPartitions.java

```diff
@@ -37,6 +37,10 @@
 
 /**
  * Executes an operation on a set of partitions.
+ *
+ * // TODO delete this class after code freeze, replace with {@link InvokeOnPartitionsAsync}
```

Comment:
Which code freeze? You mean in 3.12?

Reply:
Yes, i've meant "feature freeze"

Reply:
I mean, feature freeze has already happened so I guess we can delete it immediately?

Reply:
I've meant "after feature freeze", that is, after feature freeze is over.